src/widgets/editor.js

```diff
@@ -8,6 +8,10 @@ export default class EditorWidget extends AuthenticatedWidget {
                 save: 'onSave',
                 run: 'onRun',
             },
+            hotkeys: {
+                'Mod-Enter': e => this.run(),
+                'Mod-s': e => this.save(),
+            },
             ...options
```

Comment:
This, my friend, is the object spread operator in action.

On Tue, May 24, 2016 at 5:17 PM Tomasz Janczuk <notifications@github.com>
wrote:

> In src/widgets/editor.js
> <https://github.com/auth0/webtask-widget/pull/42#discussion_r64471300>:
>
> > @@ -8,6 +8,10 @@ export default class EditorWidget extends AuthenticatedWidget {
> >                  save: 'onSave',
> >                  run: 'onRun',
> >              },
> > +            hotkeys: {
> > +                'Mod-Enter': e => this.run(),
> > +                'Mod-s': e => this.save(),
> > +            },
> >              ...options
>
> What on earth is that? ...options?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/auth0/webtask-widget/pull/42/files/92fd92e40e483b381d6fa9b1f0ced8a9544f9bd0#r64471300>
>